sdk/schemaregistry/schema-registry/src/index.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export * from "./generated/models";
+export { SchemaRegistryClient } from "./generated/schemaRegistryClient";
```

Comment:
the generated client should be wrapped in a convenience layer with telemetry, reference doc comments, etc.

Reply:
Stopped exporting generated code